pages/index.js

```diff
@@ -204,6 +202,21 @@ export default function Home() {
           height: 1em;
         }
 
+        .button-primary {
+            color: white;
+            background-color: rgba(77,208,225, 1);
+            border-radius: 10px;
+            padding: 10px;
+            font-size: 1.5rem;
+        }
+
+        .button-primary:active,
+        .button-primary:focus,
+        .button-primary:hover {
+          background-color: rgba(77,208,225, .75);
```

Comment:
good solution for now, but something we may want a designer (i.e. Van) to take a second look at and chime in how we should deal with the hover interaction